src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -92,5 +109,27 @@ public void PEBinary_TryLoadPdbFromSymbolFolder()
                 peBinary.PdbParseException.Should().BeNull();
             }
         }
+
+        private static bool IfContainsLanguagueCode(string fileName, Language language)
```

Comment:
>IfContainsLanguagueCode [](http://example.com/codeflow?start=28&length=23)

This name is confusing. I would expect either ContainsLanguageCode or IfContainsLanguageCode<DoSomeAction> for a method name.

Reply:
In general, you can simply use 'Contains' for these, we generally don't burn 'If' into this sort of boolean test.

Look at the way the code reads, do you see the duplication of 'if'?

if (IfContainsLanguageCode...

Reply:
if (IfContainsLanguageCode...
 ---- thanks for the example. fixed.